<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MysqlToStructWidget</class>
 <widget class="QWidget" name="MysqlToStructWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>存储路径(必填)：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="editPassword">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>h5s/X_7FLkzj</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="editDbName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>das</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>root(必填)：</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>连接mysql并生成struct</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>password(必填)：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="selectDir">
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="editPath">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>/Users/zhangmengmeng/Downloads/temp</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_8">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>12</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>默认名称：mysqlToStruct.go</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="editIp">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>47.103.115.252</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>ip(必填)：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="editFileName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>dbName(必填):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>文件名称：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="editRoot">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>root</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QPushButton" name="btnDown">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>下载</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
